src/lib/Form/Type/ObjectState/ContentObjectStateUpdateType.php

```diff
@@ -70,7 +70,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     return array_filter(
                         $this->objectStateService->loadObjectStates($objectStateGroup),
                         function (ObjectState $objectState) use ($contentInfo, $contentState) {
-                            return $this->permissionResolver->canUser('state', 'assign', $contentInfo, [$objectState]) && $contentState->id !== $objectState->id;
```

Comment:
@pawbuj I'm wondering why you had to remove ` && $contentState->id !== $objectState->id`?

Reply:
there is an additional requirement ordered by @SylvainGuittard to display a list of all states

Reply:
That is the main goal of this PR. 
Users were confused to not see all the object states in the dropdown list. So we display all the states that the user has the rights to change